config/datastores/posixDatastore.yaml

```diff
@@ -0,0 +1,5 @@
+datastore:
+  cls: lsst.daf.butler.datastores.posixDatastore.PosixDatastore
+  records:
+    table: PosixDatastoreRecords
+  create: true
```

Comment:
I'm a bit surprised to not see the default mapping from StorageClass to Formatter here.  Those are specific for PosixDatastore, right?

I don't actually care if they go here or elsewhere, but we need an easy way to construct a Butler that knows about those defaults.

Reply:
I wasn't entirely sure what defaults would mean since the formatters really do lock in the file format, but you are right that FITS is where we are so our default for "how to write an Exposure" should be in this file.